### PR TITLE
Add installation instructions for vim-plug

### PR DESCRIPTION
**Pull request by AbhinavOmprakash**
I used vim-plug to install this plugin. It's much easier to use a plugin manager, lowers the barrier for entry :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,14 @@ Default shorcuts:
...
 First of all, you will need to have PostgreSQL installed and `psql` command in the `$PATH`.
 
-Then just clone this repo into your Neovim package folder:
+## Using Vimplug
+
+```
+Plug 'mzarnitsa/psql'
+```
+
+## Using git
```
Comment (mzarnitsa) [inline README.md]: Thanks for the PR!

Can you please change this line to say `Using NeoVim's native package system`?

Comment (mzarnitsa) [inline README.md]: Thanks for the PR!

Can you please change this line to say `Using NeoVim's native package system`?

Comment (AbhinavOmprakash) [reply README.md]: yes, will do

Comment (AbhinavOmprakash) [reply README.md]: I've made the change

**Review by mzarnitsa — CHANGES_REQUESTED** [review]
Hi again. Sorry, I think it's the vice versa. The way native package system works in NeoVim and Vim is one needs to copy the plugin files into `~/.config/nvim/pack/packages/start/` folder. I'm using `git` command just so one doesn't need to download/unzip/copy plugin code manually. Just one command.

What you are proposing is another simple way to install plugins, [VimPlug](https://github.com/junegunn/vim-plug). The only drawback is one needs [to install](https://github.com/junegunn/vim-plug#installation) it first, but it's also very simple process.

So, I'm suggesting to make the change something like this.

```
## Using NeoVim's native package system

Clone this repo into your Neovim package folder:
`` `
mkdir -p ~/.config/nvim/pack/packages/start/
git clone https://github.com/mzarnitsa/psql.git ~/.config/nvim/pack/packages/start/psql
`` `

## Using [VimPlug](https://github.com/junegunn/vim-plug)
`` `
Plug 'mzarnitsa/psql'
`` `
```

**Comment by AbhinavOmprakash** [comment]
HI, sorry for the delay. I've made the changes.

**Review by mzarnitsa — APPROVED** [review]
Hey! Sorry about the delay from my side. Looks good, and thank you!

**Pull request merged**